transports/quic/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "libp2p-quic"
+version = "0.8.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+description = "TLS based QUIC transport implementation for libp2p"
+repository = "https://github.com/libp2p/rust-libp2p"
+license = "MIT"
+
+[dependencies]
+if-watch = "1.1.1"
+libp2p-core = { version = "0.35.0", path = "../../core" }
+quinn = { version = "0.8.0", git = "https://github.com/kpp/quinn.git", branch = "parity_master", features = ["tls-rustls", "futures-io", "futures-core", "runtime-async-std"] }
+futures = "0.3.21"
+thiserror = "1.0.26"
+tracing = "0.1"
+rand = "0.8.5"
+
+oid-registry = "0.6.0"
+webpki = "0.22.0"
+rcgen = "0.9.2"
+ring = "0.16.20"
```

Comment:
Explicit dependency is not necessary, it is pulled by rustls

Reply:
```
error[E0432]: unresolved import `ring`
   --> transports/quic/src/tls/certificate.rs:342:13
    |
342 |         use ring::signature;
    |             ^^^^ use of undeclared crate or module `ring`

```